Packs/Zendesk/Playbooks/playbook-Zendesk_-_Ticket_Management.yml

```diff
@@ -0,0 +1,830 @@
+                root: inputs.Action
+            iscontext: true
+          right:
+            value:
+              simple: NewTicket
+          ignorecase: true
+      - - operator: isEmpty
+          left:
+            value:
+              complex:
+                root: inputs.ZendeskTicketID
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 760,
+          "y": 600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "11_18_#default#": 0.1,
+      "20_18_#default#": 0.15
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1365,
+        "width": 1870,
+        "x": -30,
+        "y": 220
+      }
+    }
+  }
+inputs:
+- key: ZendeskTicketID
+  value: {}
+  required: false
+  description: The ticket ID.
+  playbookInputQuery:
+- key: ZendeskAssigne
+  value: {}
+  required: false
+  description: The agent currently assigned to the ticket.
+  playbookInputQuery:
+- key: ZendeskCollaborators
+  value: {}
+  required: false
+  description: The users currently CC'ed on the ticket.
+  playbookInputQuery:
+- key: description
+  value: {}
+  required: true
+  description: The ticket description.
+  playbookInputQuery:
+- key: ZendeskPriority
+  value:
+    simple: high
+  required: false
+  description: The urgency with which the ticket should be addressed. Allowed values are "urgent", "high", "normal", or "low".
+  playbookInputQuery:
+- key: ZendeskRequester
+  value: {}
+  required: true
+  description: The user who requested this ticket.
+  playbookInputQuery:
+- key: ZendeskStatus
+  value: {}
+  required: false
+  description: The state of the ticket. Allowed values are "new", "open", "pending", "hold", "solved", or "closed".
+  playbookInputQuery:
+- key: ZendeskSubject
+  value: {}
+  required: true
+  description: The value of the subject field for this ticket.
+  playbookInputQuery:
+- key: ZendeskTags
+  value: {}
+  required: false
+  description: The array of tags applied to this ticket.
+  playbookInputQuery:
+- key: ZendeskType
+  value: {}
+  required: true
+  description: The type of this ticket. Allowed values are "problem", "incident", "question", or "task".
+  playbookInputQuery:
+- key: CommentToAdd
+  value: {}
+  required: false
+  description: Comment for the ticket.
+  playbookInputQuery:
+- key: addCommentPerEndpoint
+  value:
+    simple: "True"
+  required: false
+  description: 'Whether to add a new comment to the ticket for each endpoint in the incident. Possible values: True/False.'
+  playbookInputQuery:
+- key: Action
+  value: {}
+  required: false
+  description: 'Whether to open a new ticket or to add a new comment. Possible values: NewTicket/AddComment.'
+  playbookInputQuery:
+outputs:
+- contextPath: Zendesk.Ticket.id
```

Comment:
ServiceNow "id" is in capitalized letters, can you do the same here and make the relevant changes?